### ISSUE TITLE
Register applicant form - aria-label not needed

### ISSUE THREAD

**Issue by kimrm** [NoroffFEU/agency.noroff.dev#853]
The input fields that already has a `Label` element associated using the `for` tag does not need `aria-label`. Aria-label should be removed in these cases.

Example: 
```
<label class="mb-2" for="fullName">Full Name*</label>
<input
   type="text"
   class="form-control"
   aria-label="Full name"
   id="fullName"
   required
   pattern="^[a-zA-ZæøåÆØÅ]+( [a-zA-ZæøåÆØÅ]+)+$"
   title="Full Name must only contain lowercase and uppercase letters. Example: John Johnson"
/>
```

**Comment by LineNil** [comment]
Removed and added aria-label where needed.

#925 

I removed some aria-labels where there was already a for tag that described the content well enough. I also added a couple of aria-labels where they were needed.

**Comment by kimrm** [comment]
Approved and merged.
